Automattic/wp-calypso
# Gutenberg: Include App Banner

#### Changes proposed in this Pull Request

Includes the App Banner in Gutenberg

<img width="362" alt="Screenshot 2019-07-10 at 22 01 43" src="https://user-images.githubusercontent.com/43215253/61004897-e8d46e00-a35e-11e9-9313-36daa1c6e9aa.png">

#### Testing instructions

#16124 describes getting the banner to appear more eloquently than I could, but full disclaimer, I just removed the bit of logic and I'm not exactly sure how to test to make sure it leads to the right place. Would appreciate verification that it does. :)

Fixes #34460
